src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxy.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Reflection
+{
+    /// <summary>
+    /// DispatchProxy provides a mechanism for the instantiation of proxy objects and handling of
+    /// their method dispatch.
+    /// </summary>
+    public abstract class DispatchProxy
+    {
+        protected DispatchProxy()
+        {
+#if NETSTANDARD2_0
+            throw new PlatformNotSupportedException();
```

Comment:
This is still incorrect. Either remove the PNSEs (as the `System.Reflection.Emit` package already throws PNSEs) or add the string resource to it: `SR.PlatformNotSupported_ReflectionDispatchProxy`

Reply:
Pushed a new commit. Is it good now?